roads.mss

```diff
@@ -2667,6 +2667,428 @@ come in as well.
   }
 }
 
+#roads_high::surface[zoom>=16],
+#tunnel::surface[zoom>=16],
+#bridge::surface[zoom>=16] {
+  // Don't draw something for good surface, ok for racing bikes (<=28mm tyres).
+
+  // Surface ok for treking/gravel/cyclocross/city bike (28mm < tyres <= 40mm).
+  [surface='concrete:lanes'],
+  [surface='concrete:plates'],
+  [surface='paving_stones'],
+  [surface='sett'],
+  [surface='unhewn_cobblestone'],
+  [surface='cobblestone'],
+  [surface='wood'],
+  [surface='compacted'],
+  [surface='fine_gravel'],
+  [surface='woodchips'],
+  [tracktype='grade2'],
+  [tracktype='grade3'],
+  [smoothness='intermediate'],
+  [smoothness='bad'] {
+    [zoom=16] {
+      line-dasharray: 1,4;
+    }
+    [zoom=17] {
+      line-dasharray: 2,8;
+    }
+    [zoom>=18] {
+      line-dasharray: 4,16;
+    }
+
+    /* -- colors & styles -- */
+    line-color: darken(@standard-fill,@surfaceLighterDarker1);//Darken since its white.
+    [type='motorway'],
+    [type='motorway_link'] {
+      line-color: lighten(@motorway-fill,@surfaceLighterDarker1);
+    }
+    [type='trunk'],
+    [type='trunk_link'] {
+      line-color: lighten(@trunk-fill,@surfaceLighterDarker1);
+    }
+    [type='primary'],
+    [type='primary_link'] {
+      line-color: lighten(@primary-fill,@surfaceLighterDarker1);
+    }
+    [type='secondary'],
+    [type='secondary_link'] {
+      line-color: lighten(@secondary-fill,@surfaceLighterDarker1);
```

Comment:
Not sure this is visible enough on secondary (cf picture on "Boulevard d'Alsace Lorraine" you sent me). No other concerns with the colors so far.